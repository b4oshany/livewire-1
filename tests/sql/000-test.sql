\unset ECHO
\i test_setup.sql

/*******************************************************

Test Suite for LiveWire


*******************************************************/


SELECT plan(1);

SELECT has_function('lw_initialise', ARRAY['text', 'integer', 'double precision'], 'Check for lw_initialise');
SELECT * FROM finish();

ROLLBACK;
