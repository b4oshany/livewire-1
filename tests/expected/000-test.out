\unset ECHO
1..1
ok 1 - Check for lw_initialise
